src/base/utils/fs.cpp

```diff
@@ -313,7 +313,7 @@ bool Utils::Fs::isNetworkFileSystem(const QString &path)
     if (statfs(file.toLocal8Bit().constData(), &buf) != 0)
         return false;
 
-#ifdef Q_OS_MAC
+#if defined (Q_OS_MAC) || defined (Q_OS_OPENBSD)
```

Comment:
I hate being pedantic but remove spaces before the opening parentheses.

Reply:
No, shame on me for not seeing that.
Fixed.